### DISCUSSION TITLE
support for LD2450

### DISCUSSION THREAD

**Discussion by killerQueen69** [arendst/Tasmota#22091] [Support request] [answered]
Hello,
I Don't know if there is support for HLK-LD2450 sensor in the latest Tasmota. can anyone help
Thank you.

**Comment by sfromis** [comment, marked as answer]
Not on the list: https://tasmota.github.io/docs/Supported-Peripherals/ (But LD2410 is there)

**Comment by uwe5** [comment]
see https://github.com/uwe5/Tasmota-ld2450

**Reply by killerQueen69** [reply]
you are the GOAT

**Reply by jupass2612** [reply]
i have a problem i cant build for esp32

**Reply by techman83** [reply]
I cherry picked everything against the current development branch, along with fixing all the merge conflicts on a branch here -> https://github.com/techman83/Tasmota/tree/ld2450

Compiles and appears to work, though I've not done any significant testing yet. I've not read up on the requirements for adding a new sensor for a while, but it looks like with a little work it could be brought into the project.
